src/test/java/com/google/cloud/spanner/jdbc/JdbcStatementTest.java

```diff
@@ -219,6 +255,20 @@ public void testExecuteWithDdlStatement() throws SQLException {
     assertThat(statement.getUpdateCount()).isEqualTo(JdbcConstants.STATEMENT_NO_RESULT);
   }
 
+  @Test
+  public void testExecuteWithDmlReturningStatement() throws SQLException {
+    Statement statement = createStatement();
+    boolean res = statement.execute(getDmlReturningSql());
+    assertThat(res).isTrue();
```

Comment:
nit: I thought we were keeping away from `assertThat` usage.

Reply:
Changed to `assertTrue`.